frontend/src/apiAccess/apis.js

```diff
@@ -383,12 +383,17 @@ async function activityLogDayCountsPage({
 
 async function activityLogEntries({
   eventTypeName,
+  timestamp,
   effectedUser,
   actorUser,
+  actorUserEmail,
+  effectedUserEmail,
   object1ContentType,
   object1Id,
   object2ContentType,
   object2Id,
+  object1Summary,
+  object2Summary,
   page,
 }) {
```

Comment:
Everything seems to be working as before... just have one question, are we no longer filtering by the new added fields? if not should they not be removed since they're not being used inside that function?